src/core/AutoRest.Core/Properties/Resources.Designer.cs

```diff
@@ -137,22 +137,20 @@ public class Resources {
         ///   Looks up a localized string similar to Plugins:
         ///  CSharp:
         ///    TypeName: PluginCs, AutoRest.CSharp
+        ///  JsonRpcClient:
```

Comment:
Not related to my change. But since this file is autogenerated I suspect this has something to do with latest change in the master branch